src/main/java/flaxbeard/immersivepetroleum/common/blocks/tileentities/AutoLubricatorTileEntity.java

```diff
@@ -242,7 +242,7 @@ public void tick(){
 					TileEntity master = handler.isPlacedCorrectly(this.world, this, this.facing);
 					if(master != null && handler.isMachineEnabled(this.world, master)){
 						this.count++;
-						handler.lubricate(this.world, this.count, master);
+						handler.lubricate(this.world, this.count, master, this.tank.getFluid());
```

Comment:
`this.tank.getFluid().copy()` would've been better (avoids exploits) but that'll be something i can change after merging =P

---

Also, just a heads up.. i am likely gonna change a bit more on the API side of things to be sort of similar (up to a certain point) to what i have going on in `rewrite/reservoir` branch.
No, this won't mean your changes here would be in vain, i'd incorperate it/them. It's just something to look out for

Reply:
Alr, ill have to change that elsewhere in my own code, didnt realize that would be an issue. Also sounds good.